docs/sphinx/using/backends/simulators.rst

```diff
@@ -42,24 +65,64 @@ To execute a program on the :code:`nvidia` target, use the following commands:
 .. _nvidia-fp64-backend:
 
 By default, this will leverage :code:`FP32` floating point types for the simulation. To 
-switch to :code:`FP64`, specify the :code:`--target-option fp64` `nvq++` command line option for `C++` or 
-use `cudaq.set_target('nvidia', option='fp64')` for Python instead. 
+switch to :code:`FP64`, specify the :code:`--target-option fp64` `nvq++` command line option for `C++` and `Python` or 
+use `cudaq.set_target('nvidia', option='fp64')` for Python in-source target modification instead. 
+
+.. tab:: Python
+
+    .. code:: bash 
+
+        python3 program.py [...] --target nvidia --target-option fp64
+
+    The precision of the :code:`nvidia` target can also be modified in the application code by calling
+
+    .. code:: python 
+
+        cudaq.set_target('nvidia', option='fp64')
+
+.. tab:: C++
+
+    .. code:: bash 
+
+        nvq++ --target nvidia --target-option fp64 program.cpp [...] -o program.x
+        ./program.x
 
 .. note:: 
 
   This backend requires an NVIDIA GPU and CUDA runtime libraries. If you do not have these dependencies installed, you may encounter an error stating `Invalid simulator requested`. See the section :ref:`dependencies-and-compatibility` for more information about how to install dependencies.
 
+In the single-GPU mode, the :code:`nvidia` target provides the following environment variable options.
+
+.. list-table:: **Environment variable options supported in single-GPU mode**
+  :widths: 20 30 50
+
+  * - Option
+    - Value
+    - Description
+  * - ``CUDAQ_FUSION_MAX_QUBITS``
```

Comment:
Is this in addition to, or is it replacing, `CUDAQ_MGPU_FUSE`? That environment variable was advertised in a published blog with the 0.7.1 release, so unless there is a pressing need to deprecate it, I would recommend we retain that same name.

Reply:
It's in addition to (as an alias since the single-GPU simulator, i.e., no MPI required, now also supports gate fusing). 

Thanks for the information. Yea, we should retain that exact config name in this case: I added `CUDAQ_MGPU_FUSE` to the table for the multi-node section and revert the performance tuning note below to be the same as before.